third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/transport/config.go

```diff
@@ -322,6 +322,30 @@ type Config struct {
 	RequestBatcherIam          *RequestBatcher
 }
 
+func (c *Config) Clone() *Config {
```

Comment:
Does this really need to be in the third_party directory?  Could we maybe move it into the controller, and have a function like `CloneAndChangeContext`, and in the comment for that we say what it is doing, how this is sort of a hack, but it's a reasonable thing to do given what we have to work with.